language-server/modules/langserver-commons/src/main/java/org/ballerinalang/langserver/commons/completion/LSCompletionItem.java

```diff
@@ -27,4 +27,20 @@
 public interface LSCompletionItem {
 
     CompletionItem getCompletionItem();
+    
+    CompletionItemType getType();
+
+    /**
+     * Represents the completion item type.
+     * 
+     * @since 2.0.0
+     */
+    enum CompletionItemType {
+        OBJECT_FIELD,
```

Comment:
At the semantic API level, we have object fields and class fields. Just wondering whether we need that separation here as well?

Reply:
This will be used in combination with the symbol kind when we need to specifically sort the completion items. Therefore we did not add the Class field specifically. If in any case we have a requirement to get this done, let's add it. WDYT? :)